### ISSUE TITLE
stealer 

### ISSUE THREAD

**Issue by GhostyyDevv** [Im4wasp/W4SP-Stealer-Sourcecode#6]
the bot cant do >gen (user) it gives a error and can u maybe help me with obfuscating it? (Ghostyy#3056)
<img width="713" alt="sda" src="https://user-images.githubusercontent.com/126360308/221557007-2e0c376a-1a98-4c5d-a42d-8cc9197b233e.png">

**Comment by GhostyyDevv** [comment]
<img width="555" alt="Schermafbeelding 2023-02-27 132509" src="https://user-images.githubusercontent.com/126360308/221563315-6df9ff5f-bd32-431f-bb32-defc70c8ca18.png">
if i obfuscate it looks like this lol

**Comment by GhostyyDevv** [comment]
it needs a database ig

**Comment by Im4wasp** [comment]
> I have the same issue

Nah, u just didn't activate the MESSAGE CONTENT INTENT on discord dev portal 💀

**Comment by loTus04** [comment]
> the bot cant do >gen (user) it gives a error and can u maybe help me with obfuscating it? (Ghostyy#3056)

add me on discord, i'll help u

**Comment by GhostyyDevv** [comment]
> > the bot cant do >gen (user) it gives a error and can u maybe help me with obfuscating it? (Ghostyy#3056)
> 
> add me on discord, i'll help u

tag?

**Comment by GhostyyDevv** [comment]
> > I have the same issue
> 
> Nah, u just didn't activate the MESSAGE CONTENT INTENT on discord dev portal 💀

i did
but now the api is fucked up idk how to use that

**Comment by GhostyyDevv** [comment]
btw the first part needs 
intents = discord.Intents.all()

bot = commands.Bot(command_prefix='>',intents=intents) 

this in it
